BDArmory/Weapons/Missiles/BDModularGuidance.cs

```diff
@@ -888,14 +888,18 @@ private Vector3 OrbitalGuidance()
         private void UpdateOrbitalStage()
         {
             // Update list of engines/thrusters
-            engines = VesselModuleRegistry.GetModuleEngines(vessel);
-            rcsThrusters = VesselModuleRegistry.GetModules<ModuleRCSFX>(vessel);
+            engines = VesselModuleRegistry.GetModuleEngines(vessel).ToList();
```

Comment:
There's no need to add `.ToList()` (which requires a GC allocation) if the list and the entries aren't being modified.